.github/workflows/update-events.yml

```diff
@@ -2,7 +2,7 @@ name: "Actualizar eventos meetup"
 
 on:
   schedule:
-    - cron: "0 12 * * 6" # Todos los sabados a las 12:00
+    - cron: "0 17 1,2,3,4,5,6,7 * 3" # El primer miercoles del mes a mediodia UTC-5 Github esta en formato UTC
```

Comment:
Aquí dice que se va a ejecutar 8 días incluyendo el miércoles (el 1ero de cada mes AND el 2do de cada mes AND el 3ero de cada mes AND .... AND miércoles)

Reply:
En realidad es uno de esos 7, el que caiga miércoles, en el ejemplo de la imagen sería el 6, 1 y 8 caen el mismo día de semana, por eso solo coloque hasta el 7